pkg/sentry/server/ca/bundle/bundle.go

```diff
@@ -91,13 +92,13 @@ type GenerateOptions struct {
 func Generate(opts GenerateOptions) (Bundle, error) {
 	var bundle Bundle
 
-	if opts.MissingCredentials.X509 {
+	if opts.MissingCredentials.X509 && !reflect.ValueOf(opts.X509RootKey).IsNil() {
 		if err := generateX509Bundle(opts, &bundle); err != nil {
 			return Bundle{}, fmt.Errorf("failed to generate X.509 bundle: %w", err)
 		}
 	}
 
-	if opts.MissingCredentials.JWT {
+	if opts.MissingCredentials.JWT && !reflect.ValueOf(opts.JWTRootKey).IsNil() {
```

Comment:
not crazy about this but it's the only way to check that the interface's concrete value is not nil which will happen when JWT is not enabled